core/src/test/java/org/jdbi/v3/core/SampleImmutable.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core;
+
+import org.immutables.value.Value;
+import org.jdbi.v3.core.mapper.reflect.ImmutableImplementation;
+
+@ImmutableImplementation(ImmutableSampleImmutable.class)
```

Comment:
Does the `@ImmutableImplementation` serve only to point at a generated class file?  Is there some way we can discover the class file produced instead of having to write it out in the source file, just by the `@Value.Immutable` annotation?

Reply:
working on this, but at least the `@Value.Immutable` does not contain any information about the Implementation Class, but I could search it by it's generic name.